### PR TITLE
chore(build): Add encrypted ssh key for semantic-release (again)

### PR DESCRIPTION
**Pull request by dtardoin-eb**
Alright, it took so many runs, but I think everything is set up correctly. Unfortunately I couldn't get the script to work by breaking it out or moving everything into the travis config (bash is very fickle). 

I didn't actually test the release function - but I did everything short of that, [this run](https://travis-ci.org/eventbrite/eventbrite-sdk-javascript/builds/365333498) has the command specified.

### CODE REVIEW

File: _scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Ubuntu-14.04
+
+/usr/bin/openssl aes-256-cbc -K $encrypted_d09c1c3e25e0_key -iv $encrypted_d09c1c3e25e0_iv -in git_deploy_key.enc -out /tmp/git_deploy_key -d
+/bin/chmod 600 /tmp/git_deploy_key
+/usr/bin/ssh-keygen -p -P "${SSH_PASSPHRASE}" -N "" -f /tmp/git_deploy_key
+eval "$(ssh-agent -s)"
+/usr/bin/setsid ssh-add /tmp/git_deploy_key 
+ssh-add -l
+## Deploy
+npx semantic-release
```
Comment (benmvp) [inline _scripts/deploy.sh]: We'll add our steps here right?

File: _scripts/deploy_key_decrypt.sh [hunk 2]
```diff
@@ -1,20 +0,0 @@
-#!/bin/bash 
-# Decrypt Github Deploy key for the Semantic Release package to be able to push tags 
-# Ubuntu-14.04
-
-# Make sure only the current user can read the private key-
```
Comment (benmvp) [inline _scripts/deploy_key_decrypt.sh]: Can we add back all of these comments to the new script too?

File: _scripts/deploy.sh (hunk 1 above)
Comment (dtardoin-eb) [reply _scripts/deploy.sh]: Yeah, I can remove line 12 if that's not the command you're going to run to do this.

File: _scripts/deploy_key_decrypt.sh (hunk 2 above)
Comment (dtardoin-eb) [reply _scripts/deploy_key_decrypt.sh]: Yep, done.

Comment (benmvp) [reply _scripts/deploy_key_decrypt.sh]: Thanks

**Pull request merged**